pkgs/applications/networking/mailreaders/mblaze/default.nix

```diff
@@ -1,11 +1,14 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch, libiconv, ruby ? null }:
+{ coreutils, fetchFromGitHub, fetchpatch, file, gawk, gnugrep, gnused, less,
+  lib, libiconv, makeWrapper, nano, stdenv, ruby ? null}:
```

Comment:
Conventionally, the comma goes at the start of the line, and then the `}` on a line on its own:

```nix
{ foo, bar
, baz
}:

Reply:
Thanks for your review. I'll be away for a week and will get back on this then.